Mixpanel/Mixpanel.m

```diff
@@ -50,11 +50,13 @@ @interface Mixpanel () <UIAlertViewDelegate>
 #endif
 {
     NSUInteger _flushInterval;
+    BOOL _enableABTestDesigner;
 }
 
 #if !defined(MIXPANEL_APP_EXTENSION)
 @property (nonatomic, assign) SCNetworkReachabilityRef reachability;
 @property (nonatomic, strong) CTTelephonyNetworkInfo *telephonyInfo;
+@property (nonatomic, strong) UILongPressGestureRecognizer *testDesignerGestureRecognizer;
```

Comment:
We create a reference to it so that we can disable it outside of the scope of it's creation. For instance if the user initializes the library, and we start the A/B connection, then they disable A/B designer, we need to also disable the gesture recognizer to prevent reconnects.